Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/symptoms/calendar/SubmissionSymptomCalendarFragment.kt

```diff
@@ -117,4 +117,9 @@ class SubmissionSymptomCalendarFragment : Fragment(R.layout.fragment_submission_
             }
         }
     }
+
+    override fun onResume() {
+        super.onResume()
+        viewModel.onNewUserActivity()
```

Comment:
I suppose we have to call the same in onResume of `SubmissionTestResultAvailableFragment` (and `SubmissionConsentFragment`) - see the dark blue box on Figma.